modules/claims_api/spec/sidekiq/service_base_spec.rb

```diff
@@ -41,13 +41,11 @@
     poa
   end
 
-  before do
-    @service = described_class.new
-  end
```

Comment:
yikes, this was added a year ago. Do you think it's been causing problems this whole time?

Reply:
Agreed that `let` is better here 👍

Reply:
Yes! especially  
before(:all) do
 @service = foo
end